docs/source/tutorials/regression.ipynb

```diff
@@ -80,7 +80,7 @@
     "You can use `pip` to install all packages required for this tutorial as follows:\n",
     "\n",
     "```ipython3\n",
-    "!pip install scikit-learn\n",
+    "!pip install scikit-learn matplotlib==3.6.0\n",
```

Comment:
```suggestion
    "!pip install scikit-learn matplotlib>=3.6.0\n",
```

I don't think we need a hard pin to `3.6.0` ? Just minimum version required (also inconsistent with `dependencies` list below)